workload/src/main/kotlin/examples/util/LogUtil.kt

```diff
@@ -0,0 +1,14 @@
+package examples.util
+
+object LogUtil {
+
+    fun error(
+        cause: Throwable,
+        message: String? = "No message provided"
+    ) {
+        log.atError {
+            this.cause = cause
+            this.message = message
+        }
+    }
+}
```

Comment:
It would be helpful to add code demonstrating whether it works properly in subclasses and inner classes as well.